dora/tests/integration/src/test/java/alluxio/client/cli/fs/command/DoraLsCommandIntegrationTest.java

```diff
@@ -180,14 +185,14 @@ public void testLsWithSortBySizeAndReverse() throws IOException, AlluxioExceptio
   }
 
   @Test
-  public void testLsWithSortByCreationTime() throws IOException, AlluxioException {
```

Comment:
@dbw9580 

This test failure is not caused by this patch.
This test fails in main branch.
Why is it not discovered before?

Reply:
reverted. these tests have been disabled by some other PRs.